### PR TITLE
feat: encrypting unicode

### PR DESCRIPTION
**Pull request by yehjxraymond**
#4 
image

Library is not working for unicode

### CODE REVIEW

File: src/index.test.ts [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
 
 const base64Regex = /^(?:[a-zA-Z0-9+/]{4})*(?:|(?:[a-zA-Z0-9+/]{3}=)|(?:[a-zA-Z0-9+/]{2}==)|(?:[a-zA-Z0-9+/]{1}===))$/;
 const encryptionKeyRegex = new RegExp(`^[0-9a-f]{${ENCRYPTION_PARAMETERS.keyLength / 4}}$`);
 
 describe("storage/crypto", () => {
+  it("should encrypt and decrypt unicode symbols correctly", () => {
+    const originalObject = JSON.stringify({ data: "Rating(≤ 25kg)" });
+    const enc = encryptString(originalObject);
+    const dec = decryptString(enc);
+    expect(dec).toEqual(originalObject);
```
Comment (Nebulis) [inline src/index.test.ts]: Maybe can use `toStrictEqual` it's often in recommended eslint rule

File: src/index.test.ts [hunk 2]
```diff
@@ -1,9 +1,23 @@
...
+  encodeDocument,
+  decodeDocument
+} from ".";
 
 const base64Regex = /^(?:[a-zA-Z0-9+/]{4})*(?:|(?:[a-zA-Z0-9+/]{3}=)|(?:[a-zA-Z0-9+/]{2}==)|(?:[a-zA-Z0-9+/]{1}===))$/;
 const encryptionKeyRegex = new RegExp(`^[0-9a-f]{${ENCRYPTION_PARAMETERS.keyLength / 4}}$`);
 
 describe("storage/crypto", () => {
+  it("should encrypt and decrypt unicode symbols correctly", () => {
+    const originalObject = JSON.stringify({ data: "Rating(≤ 25kg)" });
```
Comment (Nebulis) [inline src/index.test.ts:16]: Do you want to test with more chars like for `encodeDocument & decodeDocument`

Comment (yehjxraymond) [reply src/index.test.ts:16]: Added a separate test for that

**Review by Nebulis — APPROVED** [review]

**Pull request merged**